core/templates/pages/android-page/android-page.component.ts

```diff
@@ -73,6 +77,8 @@ export class AndroidPageComponent implements OnInit, OnDestroy {
   featuresShown = 0;
   androidUpdatesSectionIsSeen = false;
   featuresMainTextIsSeen = false;
+  emailAddress = null;
+  name = null;
```

Comment:
Maybe explicitly mention the `string` types?

Reply:
Done, had to switch to empty strings as well as `null` gave typescript errors.

Reply:
Maybe `string | null`?

Reply:
Done